skbio/diversity/_driver.py

```diff
@@ -0,0 +1,304 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2013--, scikit-bio development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+from __future__ import absolute_import, division, print_function
+
+from functools import partial
+
+from scipy.spatial.distance import pdist
+import pandas as pd
+
+import skbio
+from skbio.diversity.alpha._faith_pd import _faith_pd, _setup_faith_pd
+from skbio.diversity.beta._unifrac import (
+    _setup_multiple_unweighted_unifrac, _setup_multiple_weighted_unifrac,
+    _normalize_weighted_unifrac_by_default)
+from skbio.util._decorator import experimental
+from skbio.stats.distance import DistanceMatrix
+from skbio.diversity._util import (_validate_counts_matrix,
+                                   _get_phylogenetic_kwargs)
+
+
+def _get_alpha_diversity_metric_map():
+    return {
+        'ace': skbio.diversity.alpha.ace,
+        'chao1': skbio.diversity.alpha.chao1,
+        'chao1_ci': skbio.diversity.alpha.chao1_ci,
+        'berger_parker_d': skbio.diversity.alpha.berger_parker_d,
+        'brillouin_d': skbio.diversity.alpha.brillouin_d,
+        'dominance': skbio.diversity.alpha.dominance,
+        'doubles': skbio.diversity.alpha.doubles,
+        'enspie': skbio.diversity.alpha.enspie,
+        'esty_ci': skbio.diversity.alpha.esty_ci,
+        'faith_pd': skbio.diversity.alpha.faith_pd,
+        'fisher_alpha': skbio.diversity.alpha.fisher_alpha,
+        'goods_coverage': skbio.diversity.alpha.goods_coverage,
+        'heip_e': skbio.diversity.alpha.heip_e,
+        'kempton_taylor_q': skbio.diversity.alpha.kempton_taylor_q,
+        'margalef': skbio.diversity.alpha.margalef,
+        'mcintosh_d': skbio.diversity.alpha.mcintosh_d,
+        'mcintosh_e': skbio.diversity.alpha.mcintosh_e,
+        'menhinick': skbio.diversity.alpha.menhinick,
+        'michaelis_menten_fit': skbio.diversity.alpha.michaelis_menten_fit,
+        'observed_otus': skbio.diversity.alpha.observed_otus,
+        'osd': skbio.diversity.alpha.osd,
+        'pielou_e': skbio.diversity.alpha.pielou_e,
+        'robbins': skbio.diversity.alpha.robbins,
+        'shannon': skbio.diversity.alpha.shannon,
+        'simpson': skbio.diversity.alpha.simpson,
+        'simpson_e': skbio.diversity.alpha.simpson_e,
+        'singles': skbio.diversity.alpha.singles,
+        'strong': skbio.diversity.alpha.strong,
+        'gini_index': skbio.diversity.alpha.gini_index,
+        'lladser_pe': skbio.diversity.alpha.lladser_pe,
+        'lladser_ci': skbio.diversity.alpha.lladser_ci}
+
+
+@experimental(as_of="0.4.0-dev")
+def get_alpha_diversity_metrics():
+    """ List scikit-bio's alpha diversity metrics
```

Comment:
Can you describe how this function can be used? i.e. the alpha diversity metrics listed here can be passed as `metric` to `alpha_diversity`